test/test_file.cpp

```diff
@@ -239,7 +239,12 @@ TEST(File_Offset)
 TEST(File_MultipleWriters)
 {
     const size_t count = 4096 / sizeof(size_t) * 256 * 2;
-
+#if defined(_WIN32) && defined(REALM_ENABLE_ENCRYPTION)
+    // This test runs really slow on Windows with encryption
```

Comment:
Does that mean that the write path in encryption on windows is substantially slower than linux/android/ios ?